packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalkContainer.js

```diff
@@ -72,52 +78,34 @@ function QuickTalkContainer ({
     setUserId((authorization && user) ? user.id : undefined)
```

Comment:
```suggestion
    setUserId(user?.id)
```

`authorization` isn't needed here, since you aren't making an API request with an Auth header.

Reply:
👍